docker-bake.hcl

```diff
@@ -73,6 +83,16 @@ target "buildkit-all" {
   inherits = ["buildkit", "all-arch"]
 }
 
+target "buildkit-archive" {
```

Comment:
No need for this. Buildkit target is used in one specific place in Buildkit's Dockerfile. All others would just be distraction. For building custom binaries change the args or mainline target.

Reply:
I don't think this was fixed. I think it still would build archives for buildkit target

Reply:
I had misunderstood yes indeed binaries are already included via BuildKit's Dockerfile.